src/ClientSymbols/QSettingsBasedStorageManagerTest.cpp

```diff
@@ -21,15 +22,24 @@ constexpr const char* kOrgName = "The Orbit Authors";
 
 namespace orbit_client_symbols {
 
-TEST(QSettingsBasedStorageManager, LoadAndSavePaths) {
-  QCoreApplication::setOrganizationName(kOrgName);
-  QCoreApplication::setApplicationName("QSettingsBasedStorageManager.LoadAndSavePaths");
+class QSettingsBasedStorageManagerTest : public ::testing::Test {
+ protected:
+  void SetUp() override {
```

Comment:
The gtest docs encourage to use the constructor instead of `SetUp`

Reply:
done